addon/components/stripe-element.js

```diff
@@ -68,15 +68,16 @@ export default Component.extend({
     stripeElement.on('blur',    (event) => this.sendAction('blur', stripeElement, event));
     stripeElement.on('focus',   (event) => this.sendAction('focus', stripeElement, event));
     stripeElement.on('change',  (...args) => {
-      let { error, complete } = args[0];
-      set(this, 'error', error);
+      let { error: stripeError, complete } = args[0];
```

Comment:
Reposting:

I think we should array destructure here while we have the chance.

```
let [{ complete, error: stripeError }] = args;
```